.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
.idea folder should not be committed as they are local IDE config files. Please delete the .idea folder and update the .gitignore file likewise.

Reply:
Done